.gitlab-ci.yml

```diff
@@ -82,6 +83,22 @@ integration:
         --network test
         test_integration_go
 
+### Checking checked-in generated code is up to date
+
+protogen:
+  stage: gen-code
+  image: nixos/nix
+  script:
+    # Create a list of paths to push to Cachix.  The secret signing
+    # key should not be available to the build process.
+    - ( unset CACHIX_SIGNING_KEY;
```

Comment:
You don't need `unset CACHIX_SIGNING_KEY` or the subshell, the signing key is not made available to this job anyway. The comment can be removed, too.

Reply:
Done in an upcoming commit. Thank you!